src/Tickets/Commerce/Cart/Unmanaged_Cart.php

```diff
@@ -14,65 +16,21 @@
  */
 class Unmanaged_Cart extends Abstract_Cart {
 
-	/**
-	 * @var string The Cart hash for this cart.
-	 */
-	protected $cart_hash;
-
 	/**
 	 * @var array|null The list of items, null if not retrieved from transient yet.
 	 */
 	protected $items = null;
 
 	/**
-	 * {@inheritDoc}
-	 */
-	public function has_public_page() {
```

Comment:
Methods and properties removed from this class were added to the `Abstract_Cart` class.